catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -317,6 +352,7 @@ private boolean isWhatsappInstalled() {
 	protected void onDestroy() {
 		webView.setDownloadListener(null);
 		webView.destroy();
+		//unregisterReceiver(onDownloadComplete);
```

Comment:
as far as i know every registered receiver must be unregistered (register in onCreate(), unregister in onDestroy() || register onStart(), unregister in OnStop()) but why this case cause 2 falling test cases, see the last 2 run test.
please if you have a clear answer, share it with me.

Reply:
You can just delete the receiver altogether, it works without it.

Reply:
you are right. the main point was the missing permission. tested and everything works fine.
thank you.

Reply:
sure thing! happy to help :wink: . I'll look at it again tomorrow.